swift/ql/lib/codeql/swift/generated/ParentChild.qll

```diff
@@ -5299,6 +5299,10 @@ private module Impl {
     or
     result = getImmediateChildOfVariadicSequenceType(e, index, partialAccessor)
   }
+
+  Element resolve(Element e) {
+    if e instanceof Element then result = e.(Element).resolve() else result = e
```

Comment:
This is strange (generated) QL.  I guess it's probably harmless.

Reply:
this was a temporary state when the root `Element` was marked `@ql.hideable` to allow the generation changes without functionally changing anything in the library (though this specific code would raise a warning as far as I know). In the final version this is actually
```codeql
  Element resolve(Element e) {
    if e instanceof HideableElement then result = e.(HideableElement).resolve() else result = e
  }
```
which is saner :slightly_smiling_face:

Reply:
Ah, I missed that looking commit-by-commit.  :+1:

Reply:
(and anyway, this will actually go away)